docs/Concepts/Privacy/Flexible-PrivacyGroups.md

```diff
@@ -1,23 +1,28 @@
 ---
-description: Onchain Privacy Groups
+description: Flexible Privacy Groups
 ---
 
-# Onchain Privacy Groups
+# Flexible Privacy Groups
 
-Onchain privacy groups use smart contracts to store and maintain the group membership. You can add
-and remove members to and from onchain privacy groups.
+Flexible privacy groups use smart contracts to store and maintain the group membership. You can add
+and remove members to and from flexible privacy groups.
+
+!!! tip
+    
+    Because group membership for flexible privacy groups is stored in a smart contract, flexible 
+    privacy groups are also known as onchain privacy groups. 
 
 !!! important
 
-    Onchain privacy groups are an early access feature. Do not use in production networks.
+    Flexible privacy groups are an early access feature. Do not use in production networks.
 
-    Onchain privacy groups are not supported with [multi-tenancy](Multi-Tenancy.md).
+    Flexible privacy groups are not supported with [multi-tenancy](Multi-Tenancy.md).
 
-    The onchain privacy group interfaces might change between releases. There might not be an
-    upgrade path from onchain privacy groups created using v1.4 to enable use of onchain privacy
+    The flexible privacy group interfaces might change between releases. There might not be an
+    upgrade path from flexible privacy groups created using v1.5 or earlier to enable use of flexible privacy
     group functionality in future versions.
 
-    It's not recommended to create onchain privacy groups in a chain with existing
+    It's not recommended to create flexible privacy groups in a chain with existing
```

Comment:
this content is duplicated in a different page - is there a way to reuse it?

Reply:
We haven't implemented a reuse strategy yet because there's not enough duplicated content to justify it.  ie, something to consider for the future but not right now